src/components/Button.tsx

```diff
@@ -164,21 +164,21 @@ const variantStyles: (contrast: boolean) => Record<
     focusStyles: Css.bshFocus.if(contrast).boxShadow(`0 0 0 2px ${Palette.Blue400}`).bgGray700.white.$,
   },
 
-  tertiaryDanger: {
-    baseStyles: Css.bgTransparent.red600.if(contrast).red400.$,
+  tertiaryQuaternary: {
+    baseStyles: Css.bgTransparent.gray900.if(contrast).gray400.$,
     hoverStyles: Css.bgGray100.if(contrast).bgGray700.white.$,
-    pressedStyles: Css.red800.if(contrast).bgWhite.gray900.$,
+    pressedStyles: Css.gray900.if(contrast).bgWhite.gray900.$,
     disabledStyles: Css.gray400.if(contrast).gray700.$,
-    focusStyles: Css.boxShadow(`0px 0px 0px 2px ${Palette.White}, 0px 0px 0px 4px ${Palette.Red500}`)
+    focusStyles: Css.boxShadow(`0px 0px 0px 2px ${Palette.White}, 0px 0px 0px 4px ${Palette.Gray900}`)
       .if(contrast)
-      .boxShadow(`0px 0px 0px 2px ${Palette.Red500}`).$,
+      .boxShadow(`0px 0px 0px 2px ${Palette.Gray500}`).$,
   },
 
-  danger: {
```

Comment:
Didn't realize the ticket was calling for the removal of the red `Danger` buttons - that seems like a risky change in terms of design language steering uses away from destructive actions - I'm going to start a thread in the beam channel to get more confirmation we're super sure about this removal

Reply:
"Quaternary" :rofl: